README.md

```diff
@@ -39,3 +39,11 @@ var split = require("shlex").split
 split('ls -al /')  // returns: [ 'ls', '-al', '/' ]
 split('rm -f "/Volumes/Macintosh HD"')  // returns [ 'rm', '-f', '/Volumes/Macintosh HD' ]
 ```
+
+### `shlex.join()`
```

Comment:
Can you please order the README so that it goes in alphabetical order by method name, with `join` first?